clean.cmd

```diff
@@ -0,0 +1,5 @@
+git clean -xdf
+git submodule foreach --recursive git clean -xdf
+git submodule foreach --recursive git reset --hard
+
+exit /b 1
```

Comment:
Why are we always exiting with a failure?

Reply:
opps copy/paste error.